docs/install/10-features.rst

```diff
@@ -1,23 +0,0 @@
-Manage features
-***************
-
-A default Inmanta install comes with all features enabled by default. :inmanta.config:option:`config.feature-file` points
-to a yaml file that enables or disables features. The format of this file is:
-
-.. code-block:: yaml
-
-    slices:
-        slice_name:
-            feature_name: bool
-
-Currently the following features are available:
-
- - core.server::dashboard
-
-An example feature file is:
-
-.. code-block:: yaml
-
-    slices:
-        core.server:
-            dashboard: false
```

Comment:
I think this one can be removed entirely?  as there are no features left? :p

Reply:
Good question. It's true that there are no features left for OSS, but I would only remove everything from line 13 and keep the rest. It's still nice to document that our server supports the concept of a feature. But I think it no longer makes sense to make this a part of the install documentation. Maybe we should move this to `Platform developer documentation`. @sanderr Any opinion on this?